ietf-mpls-te-types.yang

```diff
@@ -75,19 +75,16 @@ module ietf-mpls-te-types {
       enum per-flow {
         description
           "The load-balancing algorithm ensures that packets for
-           the same flow are forwarded on the same component link,
-           while packets for different flows can be forwarded on
-           different component links.
+           the same flow are forwarded on the same component link.
 
            Packets for different flows within the same LSP can be
            forwarded on different component links.";
       }
-      enum per-lsps {
+      enum per-top-label {
         description
-          "The load-balancing algorithm ensures that packets for
-           the same LSP are forwarded on the same component link,
-           while packets for different LSPs can be forwarded on
-           different component links.
+          "The load-balancing algorithm ensures that packets, which
+           are forwarding based on the same label at the top of the
```

Comment:
How about:

The load-balancing algorithm ensures incoming MPLS packets with the same top MPLS label and that egress on a on a LAG or bundled TE link get forwarded on the same component link of the LAG.

Reply:
See https://github.com/tsaad-dev/te/pull/104/commits/6565ce7d83432bd76208fa765f56cb991ee6b4fe